xml/containers-build.xml

```diff
@@ -26,32 +26,29 @@
    <title>Docker</title>
    <para>
     Docker is a system for creating and managing containers. Its core is the
-    Docker Open Source Engine&mdash;a lightweight virtualization solution to
-    run containers simultaneously on a single host. Docker containers are
-    defined using Dockerfiles. A Dockerfile describes how to assemble a private
-    filesystem for a container. The file can also contain metadata that
-    describes how to run the container. Most Dockerfiles start from a parent
-    image. For an introduction on how to create custom base &sle; Docker
-    images, see <xref linkend="cha-docker-overview"/>.
+    Docker Open Source Engine&mdash;a lightweight virtualization solution to run
+    containers simultaneously on a single host. Docker containers are built
```

Comment:
A `Dockerfile` doesn't produce a container, it produced a container image, which can be used to start a container with.
There are also alternative tools to produce images, so

```suggestion
    containers simultaneously on a single host. Docker containers can be built
```